### PR TITLE
Cleanup help to bring concise self-help message

### PR DESCRIPTION
**Pull request by harshavardhana**
## Description
Cleanup help to bring concise self-help message

## Motivation and Context
Cleanup

## How Has This Been Tested?
Manually

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by balamurugana** [comment]
```
Error running mint automation
+ mkdir -p 5788-05ff080/gopath/src/github.com/minio
+ git -C 5788-05ff080/gopath/src/github.com/minio clone --quiet https://github.com/minio/minio.git
+ git -C 5788-05ff080/gopath/src/github.com/minio/minio remote add minio https://github.com/minio/minio.git
+ git -C 5788-05ff080/gopath/src/github.com/minio/minio fetch --quiet minio pull/5788/head:pr5788
+ git -C 5788-05ff080/gopath/src/github.com/minio/minio checkout --quiet pr5788
+ GOPATH=/home/bala/mintauto/5788-05ff080/gopath
+ make -C 5788-05ff080/gopath/src/github.com/minio/minio --quiet
make: *** [checks] Error 1

```

**Review by nitisht — DISMISSED** [review]

**Comment by balamurugana** [comment]
```
Error running mint automation
+ mkdir -p 5788-3fdd7db/gopath/src/github.com/minio
+ git -C 5788-3fdd7db/gopath/src/github.com/minio clone --quiet https://github.com/minio/minio.git
+ git -C 5788-3fdd7db/gopath/src/github.com/minio/minio remote add minio https://github.com/minio/minio.git
+ git -C 5788-3fdd7db/gopath/src/github.com/minio/minio fetch --quiet minio pull/5788/head:pr5788
+ git -C 5788-3fdd7db/gopath/src/github.com/minio/minio checkout --quiet pr5788
+ GOPATH=/home/bala/mintauto/5788-3fdd7db/gopath
+ make -C 5788-3fdd7db/gopath/src/github.com/minio/minio --quiet
make: *** [checks] Error 1

```

**Review by nitisht — DISMISSED** [review]

**Comment by harshavardhana** [comment]
@krisis @nitisht PTAL

**Comment by balamurugana** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-xl.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#dce11303-3385-426d-a055-0f4309371722) |
<a name="dce11303-3385-426d-a055-0f4309371722"></a>
### 5788-111090e/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT: minikube:31276
ACCESS_KEY:      minioazure1
SECRET_KEY:      ***REDACTED***
ENABLE_HTTPS:    0
SERVER_REGION:   us-east-1
MINT_DATA_DIR:   /mint/data
MINT_MODE:       core

To get logs, run 'docker cp 887b7718d818:/mint/log /tmp/mint-logs'
(1/12) Running aws-sdk-go tests ... done in 1 seconds
(2/12) Running aws-sdk-php tests ... done in 8 minutes and 5 seconds
(3/12) Running aws-sdk-ruby tests ... done in 52 seconds
(4/12) Running awscli tests ... done in 3 minutes and 52 seconds
(5/12) Running mc tests ... done in 8 minutes and 2 seconds
(6/12) Running minio-dotnet tests ... done in 1 minutes and 5 seconds
(7/12) Running minio-go tests ... done in 2 seconds
(8/12) Running minio-java tests ... done in 35 seconds
(9/12) Running minio-js tests ... FAILED in 2 minutes and 45 seconds
{
  "name": "minio-js",
  "function": "getBucketPolicy(bucketName, cb)",
  "args": "bucketName:minio-js-test-d1a50584-2402-44c9-bdbf-4065d9bc4ca2",
  "duration": 260,
  "status": "FAIL",
  "error": "S3Error: The bucket policy does not exist at Object.parseError (node_modules/minio/dist/main/xml-parsers.js:56:11) at /mint/run/core/minio-js/node_modules/minio/dist/main/transformers.js:117:22 at DestroyableTransform._flush (node_modules/minio/dist/main/transformers.js:48:26) at DestroyableTransform.<anonymous> (node_modules/minio/node_modules/through2/node_modules/readable-stream/lib/_stream_transform.js:135:12) at finishMaybe (node_modules/minio/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:371:12) at endWritable (node_modules/minio/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:378:3) at DestroyableTransform.Writable.end (node_modules/minio/node_modules/through2/node_modules/readable-stream/lib/_stream_writable.js:356:5) at IncomingMessage.onend (_stream_readable.js:511:10) at endReadableNT (_stream_readable.js:974:12) at _combinedTickCallback (internal/process/next_tick.js:80:11) at process._tickCallback (internal/process/next_tick.js:104:9)"
}

Executed 8 out of 12 tests successfully.
```

**Review by nitisht — DISMISSED** [review]

File: cmd/gateway/sia/gateway-sia.go [hunk 1]
```diff
@@ -163,9 +163,16 @@ func (g *Sia) NewGatewayLayer(creds auth.Credentials) (minio.ObjectLayer, error)
...
-	logger.StartupMessage(colorBlue("  Sia Temp Directory:") + colorBold(fmt.Sprintf(" %s\n", sia.TempDir)))
+	// generates format string depending on the string length and padding.
+	formatStr := func(strLen int, padding int) string {
+		formatStr := fmt.Sprintf("%ds", strLen+padding)
+		return "%" + formatStr
+	}
+
+	logger.StartupMessage(colorBlue("\nSia Configuration:"))
+	logger.StartupMessage(colorBlue("  API Address:") + colorBold(fmt.Sprintf(formatStr(len(sia.Address), 7), sia.Address)) + "\n")
+	logger.StartupMessage(colorBlue("  Staging Directory:") + colorBold(fmt.Sprintf(" %s", sia.TempDir)) + "\n")
```
Comment (krisis) [inline cmd/gateway/sia/gateway-sia.go]: how is `7` arrived at? does it work for all terminal sizes?
Also, if there is only one place where `formatStr` is being called, can't we replace it with `fmt.Sprintf("%ds", len(sia.Address), 7)`?

Comment (harshavardhana) [reply cmd/gateway/sia/gateway-sia.go]: Oh we can I just generalized it. It works for most terminals starting from 80x25 . Its the same style used in our startup messages.

**Comment by harshavardhana** [comment]
updated PTAL @krisis @nitisht

**Review by nitisht — APPROVED** [review]

**Review by krisis — APPROVED** [review]

**Pull request merged**